UI/MapWnd.cpp

```diff
@@ -1400,20 +1401,31 @@ ModeratorActionSetting MapWnd::GetModeratorActionSetting() const
 bool MapWnd::AutoEndTurnEnabled() const
 { return m_auto_end_turn; }
 
+void MapWnd::PreRender() {
+    GG::Wnd::PreRender();
+    // Save CPU / GPU activity by skipping rendering when it's not needed
+    // As of this writing, the design and research screens have fully opaque backgrounds.
```

Comment:
Apologies, I missed that this was an existing comment.  Without changes impacting such behavior, any existing related issues fall outside of the scope of this PR.

Just to note:
> How would you suggest/expect this to work?

If it can be viewed at all (UI layout changes, alpha adjustment), the MapWnd should be updated.  If updates are prevented due to obstruction, preferably hide the window (I suspect this might be impractical, possibly render a new non-configurable opaque window on top)